airflow/providers/cncf/kubernetes/operators/kubernetes_pod.py

```diff
@@ -206,9 +206,10 @@ class KubernetesPodOperator(BaseOperator):
         to populate the environment variables with. The contents of the target
         ConfigMap's Data field will represent the key-value pairs as environment variables.
         Extends env_from.
+    :param base_container_name: The name of the base container in the pod. This container's logs
```

Comment:
Should we have some doc explnations about this? Mainly around what this capability brings

Reply:
I don't think the concept of 'base container' is mentioned anywhere already. It seems to just be the name given to the first container.. and is also the one that can be logged via get_logs=True. That info is now in this docstr. Anywhere else where we should mention this?